.travis.yml

```diff
@@ -58,7 +58,7 @@ env:
     # See https://wiki.haskell.org/Haskell_program_coverage
     - doCoverage=false
     # NOTE: Generation and installation of haddock API documentation
-    - doHaddock=false
+    - doHaddock=true
```

Comment:
Nice! :)

Reply:
That only enables Haddock compilation, it is not a mandatory check on the coverage, and we see that Hackage packaging still broken.

Actually, when I test full packaging - it falls with a very weird Nix-related error.

We also actually need update Nix submodule in itself, but that is future work.

Reply:
Sadly `doHaddock` fails the builds on macOS also.

It enables `tests` and test `eval-okay-eq-derivations` does not pass on the macOS machine in time (2 seconds). As we know that is probably not due to machine itself as probably that is due to code/test ineffectiveness.